### PR TITLE
Update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by stevenlele**
I have updated the Simplified Chinese translation.

Please note that I added several more strings at the end of the file compared to the latest template.
They are:
```
Private=
ON=
OFF=
Big Toolbar=
```
I tested and they work in the current version.
Please let me know if you'd like to add them to the template so that I can sync the line order.

### CODE REVIEW

**Comment by stevenlele** [comment]
@PeterDaveHello I'm afraid I can't find anyone to review this. I tested the translation myself and I'm sure the changes are legit. If you find some of the changings confusing, please list them and I'll explain the reason.

File: lang/transgui.zh [hunk 1]
```diff
@@ -29,12 +29,12 @@ Are you sure to remove torrent '%s' and all associated DATA?=确定要删除 Tor
...
 b=b
-Cancel=关闭
+Cancel=取消
 Client=客户端
 Close to tray=关闭到托盘图标
-Comment=评论
+Comment=注释
 Completed on=完成时间
-Completed=完成
+Completed=进度
```
Comment (PeterDaveHello) [inline lang/transgui.zh:37]: @stevenlele could you please confirm this change?

Comment (stevenlele) [reply lang/transgui.zh:37]: Yep. This is the header of the file download progress bar. Therefore, it is displayed alone, without any suffix. You know, the word “完成” or “已完成” (even this one is better) alone means “fully completed”, which does not match the meaning of “completed xx%”. In other words, Chinese users tend to not connect the header with the content. For example, “Completed on: Jan 1, 2000” (a full sentence) turns into “Completion time: Jan 1, 2000” (something: something) in Chinese. That's why I changed the translation to (download) “progress” ([下载]进度).

Comment (PeterDaveHello) [reply lang/transgui.zh:37]: Thanks, let's give it a try!

**Pull request merged**